xknx_v2.yaml

```diff
@@ -8,7 +8,10 @@ multicast_group: '224.1.2.3'
 multicast_port: 1337
```

Comment:
This is the config schema for the new config. You should touch the xknx.yaml if at all.

Reply:
v1 condif file updated with all cases of connections.